src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -1889,6 +1918,57 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 
 	//#endregion
 
+	//#region Editor Actions
+
+	getEditorActions(updateToolBar: () => void): { primary: IAction[]; secondary: IAction[] } {
```

Comment:
The `updateToolBar` is strange, how about `getEditorActions` returning an object that also includes an event when actions change?

Reply:
I'm also not happy with `updateToolBar`. Returning an event sounds good!